sisyphus/ci.go

```diff
@@ -77,8 +77,10 @@ type ProwAccessor struct {
 	prowProject   string
 	prowZone      string
 	gubernatorURL string
+	gcsBucket     string
 	gcsClient     u.IGCSClient
 	rerunCmd      func(node string) error
+	presubmitJobs []string
```

Comment:
use a map for this

Reply:
Do you mean merge protectedJobs and presubmitJobs into one map such as map[string]bool where the bool indicates if a job is in presubmit, or do you mean replacing the highlighted line as `presubmitJobs map[string]bool` where we only care about the keySet, and we can do constant time constainsKey look up instead of iterating through all keys?

Reply:
I was thinking about lookup vs iterating through all keys. If you can merge presubmit and postsubmit that sounds better

Reply:
Done